hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestRegionReplicaSplit.java

```diff
@@ -109,7 +109,7 @@ public void testRegionReplicaSplitRegionAssignment() throws Exception {
         }
       }
       // There are 6 regions before split, 9 regions after split.
-      HTU.getAdmin().split(table.getName(), Bytes.toBytes(1));
+      HTU.getAdmin().split(table.getName(), Bytes.toBytes("d"));
```

Comment:
Why this change?

Reply:
Splitting with key "1" does not split hfiles. With "d", hfiles will be split as well.

Reply:
can we add some assertions that the hfile split has happened?

Reply:
The latest patch addresses it, it now makes sure that every region has some hfiles (including the split daughter regions).